akka-actor-tests/src/test/java/akka/pattern/RetrySettingsTest.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2025 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.pattern;
+
+import org.junit.Test;
+import org.scalatestplus.junit.JUnitSuite;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class RetrySettingsTest extends JUnitSuite {
+
+  @Test
+  public void shouldCreateRetrySetting() { //mostly for testing compilation issues
+    RetrySettings retrySettings = RetrySettings.create(2);
+    assertEquals(2, retrySettings.maxRetries());
+  }
```

Comment:
I don't expect there to be any surprises, but, make sure to cover all Java API methods here. Could be compile only and doesn't necessarily need to be actual tests. Also the actual `retry` overloads.

Reply:
added